OneCryptoPkg/OneCryptoPkg.dsc

```diff
@@ -0,0 +1,51 @@
+## @file
+# OneCryptoPkg DSC file
+#
+# This file provides library mappings for OneCryptoPkg.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[Defines]
+  PLATFORM_NAME                  = OneCryptoPkg
+  PLATFORM_GUID                  = 36470E85-36F2-4BA0-8CC8-937C7D9FF888
+  PLATFORM_VERSION               = 1.0
+  DSC_SPECIFICATION              = 0x00010005
+  OUTPUT_DIRECTORY               = Build/OneCryptoPkg
+  SUPPORTED_ARCHITECTURES        = X64
+  BUILD_TARGETS                  = DEBUG|RELEASE|NOOPT
+  SKUID_IDENTIFIER               = DEFAULT
+
+[LibraryClasses]
+  #
+  # Library mappings as specified
+  #
+  UefiBootServicesTableLib       | MdePkg/Library/UefiBootServicesTableLib/UefiBootServicesTableLib.inf
+  BaseLib                        | MdePkg/Library/BaseLib/BaseLib.inf
+  BaseMemoryLib                  | MdePkg/Library/BaseMemoryLib/BaseMemoryLib.inf
+  PrintLib                       | MdePkg/Library/BasePrintLib/BasePrintLib.inf
+  MmServicesTableLib             | MdePkg/Library/StandaloneMmServicesTableLib/StandaloneMmServicesTableLib.inf
+  StandaloneMmDriverEntryPoint   | MmSupervisorPkg/Library/StandaloneMmDriverEntryPoint/StandaloneMmDriverEntryPoint.inf
+  PcdLib                         | MdePkg/Library/BasePcdLibNull/BasePcdLibNull.inf # Required for UEFI applications - NULL implementation
+  DebugLib                       | OneCryptoPkg/Library/BaseDebugLibOneCrypto/BaseDebugLibOneCrypto.inf # OneCrypto-specific DebugLib using dependency injection
+  RngLib                         | MdePkg/Library/BaseRngLibNull/BaseRngLibNull.inf # OpensslLib needs this, but actual RNG comes through dependency injection
```

Comment:
I would move this and any OpenSSL specific dependencies to a separate section (just commented in a block is fine). Because at some point, these may be chosen to be removed if a different crypto provider is used